api/models/fixtures/agent_label.json

```diff
@@ -75,6 +75,42 @@
         },
         "Yes"
       ]
+    },
+    "RegionHasCustomTopicUpdate": {
```

Comment:
The name started by copying the `RegionHasEFS` conditional in rack.json.

I do see how the mapping config makes cleaner CF json.

@mattmanning we'd still have a conditional like `EventRuleForECS` that uses `Fn::FindInMap`.

Seems like Ami, ThirdAvailabilityZone, EventRuleForECS, EFS, and this Lambda logic could all be in one map.

Why do `AvailabilityZoneConfig` and `RegionConfig` not cover the same regions?

I'll take a stab at cleaning this up more.